samples/sample_vpp/src/sample_vpp_utils.cpp

```diff
@@ -780,12 +780,14 @@ void WipeMemoryAllocator(sMemoryAllocator* pAllocator)
 
 void WipeConfigParam( sAppResources* pResources )
 {
-
-    if( pResources->multiViewConfig.View )
+    auto multiViewConfig = pResources->pVppParams->GetExtBuffer<mfxExtMVCSeqDesc>();
+    if (multiViewConfig)
     {
-        delete [] pResources->multiViewConfig.View;
+        if (multiViewConfig->View)
```

Comment:
remove this ```if``` deleting ```nullptr``` is a valid operation which simply does nothing

Reply:
done